main.py

```diff
@@ -224,7 +224,7 @@ def _find_high_delta_routes(self):
             graph[to_id].add(from_id)
 
         results = dijkstra(graph, HOME_SYSTEM_ID, self.high_deltas)
-        # results = dijkstra(graph, HOME_SYSTEM_ID, [30000142, 30002187])
+        #results = dijkstra(graph, HOME_SYSTEM_ID, [30000142, 30002187,30002537,30002062,30002758,30002016,30000304,30003676])
```

Comment:
Comments should always have a space between the # and content being commented (not a big deal, more just an FYI)